Modules/FindCUDA.cmake

```diff
@@ -734,7 +764,9 @@ if(NOT CUDA_VERSION VERSION_LESS "5.5")
   cuda_find_library_local_first(CUDA_cudart_static_LIBRARY cudart_static "static CUDA runtime library")
   mark_as_advanced(CUDA_cudart_static_LIBRARY)
 endif()
-if(CUDA_cudart_static_LIBRARY)
+
+
+if(CUDA_USE_STATIC_CUDA_RUNTIME AND CUDA_cudart_static_LIBRARY)
```

Comment:
I finally did it in a fashion that allows the user to override it with "false". The problem here was that such an override on boolean option is indistinguashable from no override.